generators/environment/templates/bin/pre-install.sh

```diff
@@ -10,11 +10,6 @@
 # of `grunt install`.
 ##
 
-if [ ! -f src/sites/default/default.settings.php ]; then
-  echo "Please run pre-install after successfully building the docroot."
-  exit 1
-fi
```

Comment:
Even with a "hard clean" the codebase is regenerated thanks to the `grunt` command prior to `grunt install`.

I removed `build/html` and ran `./bin/start.sh` with the changes that are in place.

The build was successful.

Some output for `pre-install.sh`:
```
Registering "grunt-shell" local Npm module tasks.
Reading /var/www/node_modules/grunt-drupal-tasks/node_modules/grunt-shell/package.json...OK
Parsing /var/www/node_modules/grunt-drupal-tasks/node_modules/grunt-shell/package.json...OK
Loading "shell.js" tasks...OK
+ shell
Verifying property shell.pre-ops-dispatch exists in config...OK
File: [no files]
Options: stdout, stderr, stdin, failOnError, stdinRawMode=false, preferLocal=false, execOptions={"env":null}, gruntLogHeader=false
Command: bash bin/pre-install.sh
'src/sites/default/settings.php' already exists.

Running "drush:install" (drush) task
Verifying property drush.install exists in config...OK
```

I then put the conditional back and ran `./bin/start.sh`.

I got the same output.

Putting this conditional back so we can ensure there were no errors when the codebase is built via grunt.